.Lib9c.Tests/Action/HackAndSlashTest.cs

```diff
@@ -230,10 +252,107 @@ public void Execute(int avatarLevel, int worldId, int stageId, bool contains, bo
             Assert.Equal(info.AvatarAddress, _avatarAddress);
         }
 
+        [Theory]
+        [InlineData(4, 200)]
+        public void Execute_With_UpdateQuestList(int worldId, int stageId)
```

Comment:
It seems can be simplified by avatars previous quest list in Execute case.

Reply:
> It seems can be simplified by avatars previous quest list in Execute case.

@ipdae Could you please explain more?

Reply:
If we add test parms to Execute for this case(worldId, stageId). we just check avatars quest list is added in next state.